### ISSUE TITLE
Wide ResNet-50 pretrain model is omitted

### ISSUE THREAD

**Issue by KeCh96** [kenshohara/3D-ResNets-PyTorch#96]
Hello, thanks for your pretrain models, I use resnext-101 model in my demo and get satisfactory accuracy. But the resnext-101 model cost long time, while our demo need higher speed. My tutor suggest to use WRN; however, your [pretrain model repertory](https://drive.google.com/drive/folders/1zvl89AgFAApbH0At-gMuZSeQB_LpNP-M) seems omit WRN-50. Could you please share the model? Thanks in advance.
